tests/conftest.py

```diff
@@ -169,8 +168,8 @@ def pytest_runtest_setup() -> None:
     freezegun:
     Modified to include https://github.com/spulec/freezegun/pull/424
     """
-    pytest_socket.socket_allow_hosts(["127.0.0.1"])
-    pytest_socket.disable_socket(allow_unix_socket=True)
+    # pytest_socket.socket_allow_hosts(["127.0.0.1"])
+    # pytest_socket.disable_socket(allow_unix_socket=True)
```

Comment:
Remove comment out code

```suggestion
```

Reply:
Argh, I keep leaving that in by accident. Change reverted now.